modules/hcp-create-hc-arm64-aws.adoc

```diff
@@ -8,6 +8,8 @@
 
 You can run a hosted cluster on an ARM64 {product-title} cluster for {aws-first} by overriding the default release image with a multi-architecture release image.
 
+If you do not use a multi-architecture release image, the worker nodes in the node pool are not created and reconciliation of the node pool stops until you either use a multi-architecture release image in the hosted cluster or update the `NodePool` custom resource based on the release image.
```

Comment:
I believe "worker" is correct. I see in the SSG that "worker node" is synonymous with "compute node", so I guess either would be technically correct.

Reply:
We did a sweep of the repo a few years ago to replace master with control plane and worker with compute (and for a while had a "formerly called" thing going on). If there is a case where either "worker" or "compute" is equivalent (generally, anywhere except for referring to a `worker` role), we want to go with compute. I think there are a couple exceptions where it would _have_ to be worker, but if this isn't one, better to go the other way :)

Reply:
Good to know! I didn't know the history. Happy to change it to "compute".